assets/js/components/playlist.js

```diff
@@ -52,8 +52,19 @@ function PlaylistController($rootScope, $scope, $video, $storage, $log, $notific
     }, 1000);
   };
 
+  this.scrollToCurrentlyPlaying = function() {
+    let $list = $('#video-list');
+    let $playing = $list.find('.yellow').closest('playlistitem');
+
+    $list.animate({
+      scrollTop: $playing.prop('scrollHeight')
+    }, 1000);
+  }
+
   this.setTab = function(tab) {
     this.activeTab = tab;
+
+    window.setTimeout(() => this.scrollToCurrentlyPlaying(), 250);
```

Comment:
when you click the tab, the new set of playlistitems don't show up right away. i tried running a `$scope.digest()` to let angular handle the binding update but it complained that the digest loop was already happening so...i'm like whatevaaaa